src/data/resolvers/mutations/activityLogs.js

```diff
@@ -0,0 +1,17 @@
+import { ActivityLogs, Customers, ConversationMessages } from '../../../db/models';
+
+export default {
+  /**
+   * Add conversation message log
+   * @param {Object} root
+   * @param {Object} object2 - arguments
+   * @param {string} customerId - id of customer
+   * @param {string} messageId - id of message
+   */
+  async activitivyLogsAddConversationMessageLog(root, { customerId, messageId }) {
```

Comment:
remove this mutation file. Creating activity log is a background process. We don't need mutation